SU-HSDO/suhumsci
# STN-657:  Bug - More link in views arrow wiggle

# READY FOR REVIEW

## Summary
We identified in bug on the arrow transitions for the "more-link" arrows in views, the arrow was dipping down when transitioning to the right and back.
This bug appeared both in light mode but also using dark mode inversion, we saw it most often or more obvious when testint the dark inversion settings.  This bug also was browser-specific, only occurring in both Chrome and Edge.
This also affected the arrows using `hb-display-more-link-text` utility class on Colorful themes.

![screenshot](http://g.recordit.co/pkqPrrA7Ca.gif)

Jira Card: [#657](https://sparkbox.atlassian.net/browse/STN-657)

## Need Review By (Date)
1/14

## Urgency
medium

## Steps to Test
1. npm run test
2. To test this you will need to check out multiple locations where the bug was occurring as it wasn't consistent with every more link arrow. Some locations I saw it fairly consistently: the home page (second instance of spotlights, the one within the well), [Events](http://sparkbox-sandbox.suhumsci.loc/events) and the [News](http://sparkbox-sandbox.suhumsci.loc/news) pages.
3. These pages shouldn't have any arrow transition weirdness anymore.
![screenshot](http://g.recordit.co/QMcXRwjKD3.gif)
4. If you remove the `will-change` property from the `:after `arrow it may start occurring again, but this also might be cached for render by the browser.
5. The best way to check this would be to comment out line:161 in the `_mixins.links.scss` file to test the `:after` on Traditional and utility class and to comment out line:296 on the `:before` to test on Colorful arrows without text. 
6. Once you comment those out, locate some arrows on the pages I mentioned in step 2 and verify they have the wiggle bug.
7. After you've confirmed some bugged areas, reapply the `will-change` property via uncommenting those lines and verify it fixes the issue.
8. This should be a consistent fix for both themes, all color pairings and both the Edge and Chrome browsers.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
